examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/EventHubService.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Azure.Identity;
+using Azure.Messaging.EventHubs;
+using Azure.Messaging.EventHubs.Consumer;
+using Azure.Messaging.EventHubs.Processor;
+using Azure.Storage.Blobs;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+using Microsoft.Extensions.Options;
+using System;
+using System.Threading.Tasks;
+
+namespace WebDemoWithEventHub
+{
+    public class EventHubService
+    {
+        private Settings _settings;
+
+        private EventHubConnection _eventHubConnection;
+
+        private EventProcessorClient _processorClient;
+
+        private IConfigRefresher _configRefresher;
+
+        public EventHubService(IConfiguration config, IConfigRefresher configRefresher)
+        {
+            _configRefresher = configRefresher;
+
+            _eventHubConnection = config.GetSection("WebDemo:Connection").Get<EventHubConnection>();
+            _settings = config.GetSection("WebDemo:Settings").Get<Settings>();
```

Comment:
We shouldn't need this.

Reply:
Removed.

---
In reply to: [427595227](https://github.com/Azure/AppConfiguration/pull/329#discussion_r427595227) [](ancestors = 427595227)